src/main/java/com/greenfox/tribes1/Kingdom/Kingdom.java

```diff
@@ -4,16 +4,14 @@
 import com.greenfox.tribes1.Building.Building;
 import com.greenfox.tribes1.Resources.KingdomResource;
 import com.greenfox.tribes1.Troop.Model.Troop;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
```

Comment:
wasn't the case that we don't use lombok.* but the separate lombok.getter...etc ones?